index.html

```diff
@@ -55,6 +55,7 @@ <h2>
     <p><a href="/background/">We exist to enable other approaches.</a></p>
   </div>
 
+  
```

Comment:
Why add an empty white line here?

Reply:
It's gone now. I believe it was there to show more clearly show where the svg started.